main.go

```diff
@@ -28,6 +29,19 @@ func main() {
 	if err != nil {
 		log.Print("failed to load env file, using global env")
 	}
+	if os.Getenv("WAUTH_SIGNING_KEY") == "" {
+		log.Print("failed to import env data")
+	}
+
+	dbConnectionString := fmt.Sprintf(
+		"host=%s port=%s user=%s password=%s dbname=%s sslmode=%s",
+		os.Getenv("WAUTH_DB_HOST"),
+		os.Getenv("WAUTH_DB_PORT"),
+		os.Getenv("WAUTH_DB_USER"),
+		os.Getenv("WAUTH_DB_PASSWORD"),
+		os.Getenv("WAUTH_DB_DBNAME"),
+		os.Getenv("WAUTH_DB_SSLMODE"),
+	)
```

Comment:
Oh dear, looks like that is a breaking change to our CI setup. You'll need to login to Jenkins and update the environment file secrets to use your new format, also need to update the secret manager to also include these new environment files, but other than that looking dapper! :)

Reply:
Cheers boss